UI/Pages/LoginPage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UI.Utils;
+
+namespace UI.Pages
+{
+    internal class LoginPage :AbstractPage
+    {
+        public static readonly By usernameField = By.CssSelector("#wpName1");
+        public static readonly By passwordField = By.CssSelector("#wpPassword1");
+        public static readonly By rememberMeCheckbox = By.CssSelector("#wpRemember");
+        public static readonly By loginButton = By.CssSelector("#wpLoginAttempt");
+
+        public void EnterUsername(string username, int timeSeconds)
```

Comment:
Functions of pages need to return new PageName(). It is called Fluent Page Object. Read it please
https://www.lambdatest.com/blog/fluent-interface-design-pattern/